Mod Manager/src/routes/modList/+page.svelte

```diff
@@ -806,4 +959,36 @@
 	:global(.bx--snippet.bx--snippet--single) {
 		background-color: #262626;
 	}
+
+	.overflow-y-auto {
```

Comment:
Use Tailwind (or inline styles if there is no Tailwind class for colour scheme) for this; overriding a style like this may override it globally across all pages which is highly inadvisable.